sql/information_schema/information_schema.go

```diff
@@ -1993,6 +1991,30 @@ func partitionKey(tableName string) []byte {
 	return []byte(InformationSchemaDatabaseName + "." + tableName)
 }
 
+func getColumnDefaultString(cd *ColumnDefaultValue) interface{} {
+	if cd.String() == "" {
+		return nil
+	} else if cd.IsLiteral() && cd.String() == "NULL" {
+		return nil
+	} else if cd.IsLiteral() && cd.String() == `""` {
+		return ""
+	} else {
+		// in FromDoltSchema function, all default values are handled as expression including literal value,
```

Comment:
Remember GMS is different project than Dolt. We should should leave any comments pertaining to Dolt here.